stubs/docutils/docutils/writers/__init__.pyi

```diff
@@ -1,3 +1,28 @@
-from _typeshed import Incomplete
+from typing import Any, ClassVar, Generic, TypeVar
 
-def __getattr__(name: str) -> Incomplete: ...
+from docutils import Component, nodes
+from docutils.io import Output
+from docutils.languages import LanguageImporter
+
+_S = TypeVar("_S", bound=str | bytes)
+
+class Writer(Component, Generic[_S]):
+
+    document: ClassVar[nodes.document | None] = None
+
+    output: _S | None = None
+
+    language: ClassVar[LanguageImporter | None] = None
+
+    destination: ClassVar[Output | None] = None
+
+    parts: dict[str, Any]
+
+    def __init__(self) -> None: ...
+    def write(self, document: nodes.document, destination: Output) -> _S: ...
```

Comment:
This can return anything actually. For instance with `destination = NullOutput`, this returns None and not `str` or `bytes`.

Reply:
However, I think it's usually `str | bytes | None`, but accepting `None` is likely to add more errors.